test/e2e/tests/network/network-manager.spec.ts

```diff
@@ -16,7 +16,9 @@ describe('Network Manager', function (this: Suite) {
       {
         fixtures: new FixtureBuilder()
           .withNetworkControllerOnMainnet()
-          .withEnabledNetworks({ eip155: { '0x1': true } })
+          .withEnabledNetworks({
```

Comment:
nit - these test files seem to all be linting changes.
We can potentially revert if we want to reduce file changes.